modules/k8s/namespace.go

```diff
@@ -32,21 +32,21 @@ func CreateNamespaceE(t *testing.T, options *KubectlOptions, namespaceName strin
 
 // GetNamespace will query the Kubernetes cluster targeted by the provided options for the requested namespace. This will
 // fail the test if there is an error in creating the namespace.
```

Comment:
Can you update the docs to mention that this will also fail the test if the namespace doesn't exist?

Reply:
Sure!

Reply:
Done